scripts/run-monitor-ci-tests.bash

```diff
@@ -0,0 +1,181 @@
+		workflows_ids=( $(echo ${workflows} | jq -r '.items | .[].id') )
+
+		# report failed jobs per workflow
+		for workflow_id in "${workflows_ids[@]}"; do
+			workflow_status=$(echo ${workflows} | jq -r --arg id "${workflow_id}" '.items | map(select(.id == $id)) | .[].status')
+
+			if [[ "$workflow_status" == "success" ]]; then
+				printf "\nSUCCESS: monitor-ci workflow with id ${workflow_id} passed: https://app.circleci.com/pipelines/github/influxdata/monitor-ci/${pipeline_number}/workflows/${workflow_id} \n"
+			else
+				# set job failure
+				is_failure=1
+
+				# get the jobs that failed for this workflow
+				jobs=$(curl -s --request GET \
+					--url "https://circleci.com/api/v2/workflow/${workflow_id}/job" \
+					--header "Circle-Token: ${API_KEY}" \
+					--header 'content-type: application/json' \
+					--header 'Accept: application/json')
+
+				# print the names of the failed jobs
+				printf "\nFailed jobs:\n"
+				failed_jobs=$(echo ${jobs} | jq '.items | map(select(.status == "failed"))')
+				failed_jobs_names=( $(echo ${failed_jobs} | jq -r '.[].name') )
+				for name in "${failed_jobs_names[@]}"; do
+					printf " - ${name}\n"
+				done
+
+				# get the artifacts for each failed job
+				printf "\nArtifacts from failed jobs:\n"
+				for name in "${failed_jobs_names[@]}"; do
+					printf "\n===== ${name} =====\n"
+					job_number=$(echo ${failed_jobs} | jq -r --arg name "${name}" 'map(select(.name == $name)) | .[].job_number')
+					artifacts=$(curl -s --request GET \
+					--url "https://circleci.com/api/v1.1/project/github/influxdata/monitor-ci/${job_number}/artifacts" \
+						--header "Circle-Token: ${API_KEY}" \
+						--header 'content-type: application/json' \
+						--header 'Accept: application/json')
+
+					artifacts_length=$(echo ${artifacts} | jq -r 'length')
+					if [ ${artifacts_length} -eq 0 ]; then
+						printf "\n No artifacts for this failed job.\n"
+					else
+						artifacts_urls=( $(echo ${artifacts} | jq -r '.[].url') )
+						# print each artifact text and link
+						for url in "${artifacts_urls[@]}"; do
+							path=$(echo ${artifacts} | jq --arg url "${url}" 'map(select(.url == $url)) | .[].pretty_path')
+							printf '\n- %s\n' "${path}"
+							printf '   - URL: %s\n' "${url}"
+
+							# download artifact
+							filename=$(basename "${path}")
+							filename="${filename::-1}" # removes extra " from end
+							curl -L --output "monitor-ci/test-artifacts/results/${name}/${filename}" --request GET \
+								--url "${url}" \
+								--header "Circle-Token: ${API_KEY}"
+						done
+					fi
+				done
+
+				printf "\n\nFAILURE: monitor-ci workflow with id ${workflow_id} failed:\n"
+				printf "https://app.circleci.com/pipelines/github/influxdata/monitor-ci/${pipeline_number}/workflows/${workflow_id}"
+			fi
+		done
+
+		exit $is_failure
+	fi
+
+	# sleep 1 minute and poll the status again
+	attempts=$(($attempts+1))
+	remaining_attempts=$(($max_attempts-$attempts))
```

Comment:
what, no exponential decay on your backoff strategy? nah... thanks for adding this